### PR TITLE
Add references to `play.sbt.PlayImport`

### PR DESCRIPTION
**Pull request by jeantil**
This commit intends to make it easy for people using scala builds to find the proper import for play keys and configuration elements.

### CODE REVIEW

File: documentation/manual/releases/migration24/Migration24.md [hunk 1]
```diff
@@ -67,7 +68,7 @@ IntelliJ is now able to import sbt projects natively, so we recommend using that
 
 ### Play SBT plugin API
 
-All classes in the SBT plugin are now in the package `play.sbt`, this is particularly pertinent if using `.scala` files to configure your build.
+All classes in the SBT plugin are now in the package `play.sbt`, this is particularly pertinent if using `.scala` files to configure your build. You will need to importidentifiers from `play.sbt.PlayImport` to use play provided configuration elements.
```
Comment (mkurz) [inline documentation/manual/releases/migration24/Migration24.md]: Typo: `importidentifiers` -> `import identifiers`

**Comment by jeantil** [comment]
@mkurz  thanks for the review, I have fixed the typo and repushed the commit

**Comment by jroper** [comment]
Forward ported to master: 78ba829

**Comment by jeantil** [comment]
thanks !

**Pull request merged**